### ISSUE TITLE
Add support for indentwise plugin

### ISSUE THREAD

**Issue by danyx23** [VSCodeVim/Vim#7667]
**Is your feature request related to a problem? Please describe.**

When navigating in languages with significant whitespace (e.g. Python, F#) it would be very useful to be able to move the cursor based on indentation levels like [vim-indentwise](https://github.com/jeetsukumaran/vim-indentwise)

**Describe the solution you'd like**

Allow optional movements like vim-indentwise (i.e. jump to relative indentation levels like forward with less indent; jump to next/previous absolute level of indent; jump to begin/end of block)

**Describe alternatives you've considered**

The existing vim-indent-object handles some of the features - but e.g. in large python files it is very nice to be able to "jump up to next line with 0 indetation" (or 1 or whatever) and then just repeat the motion with ";".

**Additional context**

I created a PR for this, will be linked here in a sec

**Comment by banoris** [comment]
Anything else to do before we can merge this? Looking forward to have this, it makes navigating through Python/YAML a lot easier!

**Comment by danyx23** [comment]
@banoris I'll try to rebase on top of master this weekend and see if we can get this merged

**Comment by red8888** [comment]
@danyx23 Just curious if there are plans to merge it eventually still. Thanks!

**Comment by danyx23** [comment]
@red8888 yes from my side - I'll rebase and update in the coming days and then try to ping the maintainer again :)

**Comment by chrisshroba** [comment]
I'm also super interested in this! Is there any work I can help with?

**Comment by danyx23** [comment]
Hi everyone who is interested in this plugin (cc @chrisshroba @red8888 @banoris)! I have a bit of time in the next few days and would like to make a push to finally get this merged in the not too far future. For this it would be great if some people could test it a bit.

To this end I have updated the current PR and added a compiled version of the current version 1.26.1 plus my changed. You can find this plugin download in the [most recent comment on the PR](https://github.com/VSCodeVim/Vim/pull/7668#issuecomment-1763388149) and follow the instructions there. 

The PR also contains a new section in the readme that lists all the keyboard shortcuts.

If anyone here is familiar with developing VS code plugins you can of course also checkout this branch and compile it yourself (instructions for this are in the ./.github/CONTRIBUTING.md file of the VS Code VIM repo)

Looking forward to some feedback and fingers crossed this can be merged soon :)
